fhirpipe/extract/graphql.py

```diff
@@ -132,3 +147,12 @@ def run_graphql_query(graphql_query, variables=None):
         raise Exception(
             f"Query failed with returning code {request.status_code}\n{request.reason}."
         )
+
+
+def get_credentials(credential_id):
+
+    resp = run_graphql_query(credential_query, variables={"credentialId": credential_id})
+    credentials = resp["data"]["credential"]
```

Comment:
this will raise `KeyError` if data is null right ?

Reply:
Right, and I think the logic is wrong here, I'll change that

Reply:
actually I was the one making a mistake here, if the credential does not exist, it will return something like `{data: { credential: null} }`